src/components/HacktoberFestCall/UserProgress.js

```diff
@@ -176,11 +186,31 @@ function UserProgress({ user }) {
           </ProgressBody>
           <ProgressStatus>
             {!completed ? (
-              <ProgressStatusText>
-                Você tem{" "}
-                <strong>{progress.opened} pull request(s) enviado(s)</strong> e{" "}
-                <strong>{progress.merged} aceito(s)</strong>
-              </ProgressStatusText>
+              achievements.opened ? (
+                <ProgressStatusText>
+                  Você tem{" "}
+                  <strong>
+                    {pluralization(
+                      progress.opened,
+                      "pull request enviado",
+                      "pull requests enviados"
```

Comment:
Hi @brunoocasali.

IMHO, you are not thinking in React... What do you think about convert this function in a React Component? Such as:

```jsx
<Pluralization singular="pull request enviado" plural="pull requests enviados" />
```

In a quick search I found this react component: https://github.com/tsmith123/react-pluralize

Reply:
@marcelometal thanks for your advice! I have no "real" experience with react, I'll implement your suggestion! 👍

Reply:
Due to the size of the change I really didn't even think about using a library for this, I'm afraid to add libs to projects that I don't mantain because I don't want to flood the project's build size, but at the end using a library was a smart choice, so thanks again for the tip @marcelometal.

By the way this project has no test coverage, submitting some pull requests about this will be nice in your opinion @marcelometal ?